lib/evss/gi_bill_status/service.rb

```diff
@@ -39,6 +51,12 @@ def get_gi_bill_status
         response = OpenStruct.new(status: e.status, body: e.body)
         EVSS::GiBillStatus::GiBillStatusResponse.new(response.status, response)
       end
+
+      def self.current_time
```

Comment:
how about naming this method `get_current_time` and then bringing back the `current_time` variables to use this method? theoretically it's possible that the time will change from the start of the function to the end of the function and that would screw up `Time.new(current_time.year, current_time.month, current_time.day)`

Reply:
good idea.